deployment/terraform/.terraform/plugins/linux_amd64/lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
It would be good to ensure that these files are taken out of the branch and ignored with:

```
# Terraform
.terraform
*.tfplan
*.tfvars
*.tfstate
*.tfstate.backup
```